libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -503,9 +503,15 @@ void PbftManager::initialState_() {
 
   auto soft_voted_block_hash = db_->getPbftMgrVotedValue(PbftMgrVotedValue::SoftVotedBlockHashInRound);
   auto soft_voted_block = db_->getPbftMgrStatus(PbftMgrStatus::SoftVotedBlockInRound);
+
   if (soft_voted_block_hash) {
     // From DB
     soft_voted_block_for_this_round_ = std::make_pair(*soft_voted_block_hash, soft_voted_block);
+    for (const auto &vote : db_->getSoftVotes(round)) {
+      if (vote->getBlockHash() == *soft_voted_block_hash) {
+        vote_mgr_->addVerifiedVote(vote);
+      }
+    }
```

Comment:
If soft_voted_block_hash has value, the current round soft votes saved in soft_votes column in DB. Don't need add back to verified votes queue

Reply:
I think we need to, because votes are saved in DB but never retrieved.  Before we have also kept them in verified_votes column in DB, but now we only keep there our own votes. But maybe I do not understand whole picture and we are not using soft votes after we soft-voted block, then I think we should not save them to DB at all. What do you think @gaoqi8649 ?